### PR TITLE
Added scrollable configuration instructions

### PR DESCRIPTION
**Pull request by FelixAkk**
Implements https://github.com/jasonkriss/ember-scroll-to/issues/16#issuecomment-160713464

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,3 +81,23 @@ this.get('scroller').scrollVertical(target, options);
...
 * `complete` -- a callback to execute once the scrolling animation is complete.
+
+## Configuration
+Some frameworks - like Google's Material Design Lite - will use a custom DOM structure to wrap the main content (e.g. for facilitating responsive design, modal overlays). For use in such environment, you can override the default scrollable element (`html, body`) with the container element that should be used by the service to set the vertical scroll position. In your router code, add the following:
+```javascript
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+
+  i18n: Ember.inject.service(),
```
Comment (blimmer) [inline README.md]: I think you injected the wrong service here

Comment (FelixAkk) [inline README.md]: Oh snap, my bad! Wasn't paying enough attention when copying which injection line over. Thank you! :D

**Comment by jasonkriss** [comment]
Hey @FelixAkk. Thanks for taking the time to put this together. I think a mention in the Readme is a good way to address this. I think a more idiomatic way to do this would be to simply extend the service and override the scrollable computed property. So something like this...

``` javascript
// In app/services/scroller.js

import Scroller from 'ember-scroll-to/services/scroller';

export default Scroller.extend({
  scrollable: Ember.computed(function() {
    return Ember.$('main.mdl-layout__content');
  })
});
```

What do you guys think? @blimmer

**Comment by blimmer** [comment]
I definitely think the way @jasonkriss is recommending is the more ember-y way to do this

**Comment by FelixAkk** [comment]
Thanks @jasonkriss and @blimmer :) Still very new to Ember.js so wasn't aware of this. Updated the pull request to include it.

**Comment by jasonkriss** [comment]
Awesome! Thanks for doing that. Do you mind just squashing your commits? Then I'll get this merged in right away. Thanks!

**Comment by FelixAkk** [comment]
Squashed. Excuse me for the late reaction :)

**Comment by jasonkriss** [comment]
Awesome! Thanks @FelixAkk!

**Pull request merged**